hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/ethereum/EthTxData.java

```diff
@@ -60,6 +61,19 @@ public record EthTxData(
     static final BigInteger LEGACY_V_BYTE_SIGNATURE_0 = BigInteger.valueOf(27);
     static final BigInteger LEGACY_V_BYTE_SIGNATURE_1 = BigInteger.valueOf(28);
 
+    static final int FOUNDRY_DETERMINISTIC_DEPLOYER_GAS_PRICE_MULTIPLIER = 100;
+    // The specific transaction bytes that are used to deploy the Foundry Deterministic Deployer contract
+    public static final byte[] FOUNDRY_DETERMINISTIC_DEPLOYER_TRANSACTION;
+
+    static {
+        try {
+            FOUNDRY_DETERMINISTIC_DEPLOYER_TRANSACTION = Hex.decodeHex(
+                    "f8a58085174876e800830186a08080b853604580600e600039806000f350fe7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe03601600081602082378035828234f58015156039578182fd5b8082525050506014600cf31ba02222222222222222222222222222222222222222222222222222222222222222a02222222222222222222222222222222222222222222222222222222222222222");
```

Comment:
Can we reference https://github.com/Arachnid/deterministic-deployment-proxy?tab=readme-ov-file#deployment-transaction in a comment here?  (It's in the PR comment but that's disconnected from the source code.)

Reply:
BTW, that readme also says:
> **Note:** as of last readme update; don't trust these to be latest!
!!

Reply:
will fix in the next release.